app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb

```diff
@@ -345,7 +356,7 @@ def parse_instance(instance)
             :cpu_sockets          => flavor[:cpus],
             :cpu_total_cores      => flavor[:cpu_cores],
             :cpu_cores_per_socket => 1,
-            :memory_mb            => flavor[:memory] / 1.megabyte,
+            :memory_mb            => flavor[:memory].to_i / 1.megabyte,
```

Comment:
This and this [https://github.com/ManageIQ/manageiq/pull/8968/files/a5f5df71ee54bd6a04a2558012dc7151e76736db#diff-4004a2472d57db1b52ea4e53294be8b7R326] (this) are so that if in the future we fail to get the flavor (some new type of flavor that also isn't returned with list) refresh wont throw an exception.